styles.css

```diff
@@ -590,6 +711,126 @@ section.footer-m {
   background-size: cover;
 }
 
+/* Pop-up */
+
+section#pop-up {
+  width: 100%;
+  height: 100%;
+  background: #1a2236;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  box-sizing: border-box;
+}
+
+.pop-up-content {
+  display: flex;
+  flex-direction: column;
+  background: white;
+  height: 100%;
+  padding: 20px 20px;
+  box-sizing: border-box;
+  overflow-y: scroll;
+}
+
+.pop-up-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+}
+
+.pop-up-header .menu-btn {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.image-gallery-container {
+  display: flex;
+  flex-direction: column;
+  margin-top: 30px;
+}
+
+.selected-image-container {
+  position: relative;
+  width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.gallery {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  justify-content: flex-start;
+  margin: 0;
+}
+
+.gallery-item-container {
+  display: flex;
+  width: 25%;
+  height: auto;
+}
+
+.gallery-image {
+  width: 100%;
+  height: auto;
+  object-fit: contain;
+}
+
+.gallery-control {
+  position: absolute;
+  border: none;
+  display: flex;
+  height: 99%;
+  justify-content: center;
+  align-items: center;
+}
+
+img.gallery-icon {
+  width: 10px;
+}
+
+.prev {
+  left: 0;
+  background: linear-gradient(90deg, #0000007a, transparent);
+}
+
+.next {
+  right: 0;
+  background: linear-gradient(270deg, #0000007a, transparent);
+}
+
+.bottom-pop-up-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 700px;
+  align-self: center;
+  margin-top: 20px;
+}
```

Comment:
For the mobile version, kindly consider tweaking the margins, paddings and other dimensions to match the figma design. There are some slight variations are shown below :point_down: Please always remember that all screen widths below 768px should look just like the figma design of the mobile version :wink: 

### Your design


![Screenshot from 2022-12-21 22-50-21](https://user-images.githubusercontent.com/65233860/209019740-ade03f62-8bb5-41ac-8dd4-3a4914d65dc4.png)

![Screenshot from 2022-12-21 22-58-23](https://user-images.githubusercontent.com/65233860/209019808-a24e4d5f-441c-4b1d-b9c8-825e30f75a88.png)



### Figma design


![Screenshot from 2022-12-21 22-46-37](https://user-images.githubusercontent.com/65233860/209019750-aa3e0b4e-cd75-4d22-8e58-eb745e0303a7.png)


![Screenshot from 2022-12-21 22-58-12](https://user-images.githubusercontent.com/65233860/209019823-a27c0126-ab81-42c2-a8ef-7d985fa3cc40.png)